app/views/user_mailer/anonymous_or_unrevealed_notification.html.erb

```diff
@@ -0,0 +1,26 @@
+<% content_for :message do %>
+  <p><%= t(".greeting", user: style_bold(@user.default_pseud.byline)).html_safe %></p>
+
+  <p><%= t(".changed_status",
+           collection: style_link(@collection.title, collection_url(@collection)),
+           work: style_link(@work.title.html_safe, work_url(@work)),
```

Comment:
I think we should use `style_creation_link` here, now that we have it:  https://github.com/otwcode/otwarchive/blob/b3c9f726a879b754c84a960d1611f0037dcef0f4/app/helpers/mailer_helper.rb#L12-L16